staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_external.go

```diff
@@ -1005,7 +1005,7 @@ func (g *Cloud) firewallObject(name, region, desc string, sourceRanges utilnet.I
 	firewall := &compute.Firewall{
 		Name:         name,
 		Description:  desc,
-		Network:      g.networkURL,
```

Comment:
GIven the size of Cloud struct, I think it's quite risky that someone will read this field directly (e.g. in some other file).

Can we rename 'g.networkURL' to something like 'g.unsafeNetworkURL' with comment why reading this field directly (instead of calling g.NetworkURL) is unsafe?

Reply:
Actually - networkURL is safe to read - it's always properly initialized.

Reply:
Good point, should we revert all g.networkURL -> g.NetworkURL() migrations in this PR?